OperatingSystem-Services/Platform-Linux/OpenVAS/README.md

```diff
@@ -0,0 +1,419 @@
+                    "666"
+                    ]
+                volumes:
+                    - gpg_data_vol:/etc/openvas/gnupg
+                    - vt_data_vol:/var/lib/openvas/plugins
+                    - notus_data_vol:/var/lib/notus
+                    - ospd_openvas_socket_vol:/run/ospd
+                    - redis_socket_vol:/run/redis/
+                    - openvas_data_vol:/etc/openvas/
+                    - openvas_log_data_vol:/var/log/openvas
+                depends_on:
+                    redis-server:
+                    condition: service_started
+                    gpg-data:
+                    condition: service_completed_successfully
+                    vulnerability-tests:
+                    condition: service_completed_successfully
+                    configure-openvas:
+                    condition: service_completed_successfully
+
+                gvm-tools:
+                image: registry.community.greenbone.net/community/gvm-tools
+                volumes:
+                    - gvmd_socket_vol:/run/gvmd
+                    - ospd_openvas_socket_vol:/run/ospd
+                depends_on:
+                    - gvmd
+                    - ospd-openvas
+
+            volumes:
+                gpg_data_vol:
+                scap_data_vol:
+                cert_data_vol:
+                data_objects_vol:
+                gvmd_data_vol:
+                psql_data_vol:
+                vt_data_vol:
+                notus_data_vol:
+                psql_socket_vol:
+                gvmd_socket_vol:
+                ospd_openvas_socket_vol:
+                redis_socket_vol:
+                openvas_data_vol:
+                openvas_log_data_vol:
+
+            secrets:
+                server-certificate:
+                    file: /home/blueteam/docker_keys/servercert.pem
+                private-key:
+                    file: /home/blueteam/docker_keys/serverkey.pem
+        </details>
+
+5. **Starting The Docker Containers**
+    
+    > [!NOTE] This process could take a long time (Possibly several hours) depending on your machine's resources. Consider using tmux to run commands in order to save and come back.
+
+    **Pull (download) the docker images:**
+    ```
+    docker compose -f $DOWNLOAD_DIR/docker-compose.yml -p greenbone-community-edition pull
+    ```
+    <img  src="Images/DockerPull.png">
+
+
+    **Start the docker containers (This could take a long time):**
+    ```
+    docker compose -f $DOWNLOAD_DIR/docker-compose.yml -p greenbone-community-edition up -d
+    ```
+    <img  src="Images/DockerUp.png">
+
+    **Ensure all docker containers started:**
+    ```
+    docker ps
+    ```
+    <img  src="Images/DockerPS.png">
+
+6. **Create an admin user**
+   
+   Create an admin user account with the command:
+   ```
+   docker compose -f $DOWNLOAD_DIR/docker-compose.yml -p greenbone-community-edition \
+    exec -u gvmd gvmd gvmd --user=admin --new-password='<PASSWORD>'
```

Comment:
Might be good to add a point that they should replace <PASSWORD>. I think it is common knowledge to replace anything between the brackets but always good to be safe.